Library/MessageGenerator.swift

```diff
@@ -198,11 +198,7 @@ struct MessageGenerator {
     }
 
     func minuteString(for interval: Double) -> String {
-        let minutes = interval / 60
-
-        let formatter = NumberFormatter()
-        formatter.maximumFractionDigits = 2
-        let minutesString = formatter.string(from: NSNumber(value: abs(minutes)))!
+        let minutesString = String(describing: Int(round(interval / 60)))
```

Comment:
@marijnschilling we can add the unit test in a separate PR. 

Instead of describing I could have used `"\(Int(round(interval / 60)))"` but it's harder to read because of the escaping, and also doing it that way should be preferred when your string has a format, I don't need any of that.